CapitalGainsImporter.js

```diff
@@ -139,12 +139,15 @@ CKI.inputRowData = function(dataSource, row) {
 			var formIndex = rowResp.formIndex;
 			var rowIndex = rowResp.rowIndex;
 
-			var ht = element.querySelector('[name="capitalGains['+formIndex+'].holdingType"]');
-			ht.value = data.holdingType;
-
-			var rc = element.querySelector('[name="capitalGains['+formIndex+'].reported"]');
+			var rc = element.querySelector('[name="capitalGains['+formIndex+'].reportingCategory"]');
 			rc.value = data.reportingCategory;
 
+			var ac= element.querySelector('[name="capitalGains['+formIndex+'].adjustmentCode"]');
```

Comment:
With your change, I wasn't sure what exactly to put for "adjustmentCode" in my CSV file if I wanted to it to mean empty. (If you know, can you put it in the README.md?)

Alternatively, it may be easier to NOT add these two columns at all -- from what I have seen, these values are always empty/0.0.